ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ip_dhcp_snooping.schema.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  ip_dhcp_snooping:
+    type: dict
+    keys:
+      enabled:
+        type: bool
+      bridging:
+        type: bool
+      information_option:
+        type: dict
+        keys:
+          enabled:
+            type: bool
+            description: |
+              "Enable insertion of option-82 in DHCP request packets"
+          circuit_id_type:
+            type: str
+            convert_types:
+            - int
+            description: |
+              "< none > or <0 - 255>"
+          circuit_id_type_format:
```

Comment:
If this option only allows the two described strings, we should add `valid_values` to the schema.

Reply:
Done.